data/data_convert.py

```diff
@@ -0,0 +1,253 @@
+import copy
+import json
+import os
+from data_utils import add_current_place, add_current_objects, downsampling, get_real_task_id, compose_instance_v4_sg, compose_instance_v4_sgaslabel
+import argparse
+from collections import defaultdict, Counter
+from subgoals.sg_generating import sg_generator
+
+# arguments
```

Comment:
Do you want to put this inside a __main__? It's not required here since there is no code you would ideally want to import elsewhere, but its a good practice in general.

Reply:
No because here I'm generating the data. I want it to be separate.